doc/userdoc/contribute/nest_with_ides.rst

```diff
@@ -106,3 +106,140 @@ Usage
   ``Debug`` should also start a debugging session. To get most out of
   debugging, run ``cmake`` in the build directory with
   ``-Dwith-debug=ON``.
+
+
+Visual Studio Code
+------------------
+
+The following section will guide you through setting up Visual Studio Code (VS Code) for editing, building,
+running, and debugging NEST. Tested with Ubuntu 20.04. Steps for macOS should be equivalent, but with ``⌘``
+instead of ``ctrl`` in keyboard shortcuts.
+
+Requirements and limitations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Assumes a suitable compiler (GCC/Clang/etc.) is installed.
+* Assumes CMake version 3.15 or newer is installed.
+* C++ debugging assumes GDB is installed.
```

Comment:
In the case of Mac, Xcode should be installed which comes with the debugger `lldb`.

Reply:
Added.